src/HMRServer.js

```diff
@@ -35,6 +35,16 @@ class HMRServer {
       ws.send(msg);
     }
   }
+
+  emitReload() {
+    let msg = JSON.stringify({
+      type: 'reload'
+    });
```

Comment:
Why would you stringify a JSON literal right after creating it o_0? I’m sure it’s not a _huge_ performance penalty, but it feels a bit wrong

Reply:
We are sending the string through the socket. See line 45.

Reply:
Ya exactly, so why not just keep it a string the whole time?

So instead of:
```javascript
let msg = JSON.stringify({
    foo: 'bar'
})
```
We can do:
```javascript
let msg = `{
    foo: ‘bar’
}`
```

You get me? Like in the original, we are creating a new JavaScript object using JSON literal notation, and then passing it into `JSON.stringify` as a parameter, and then taking the outputted string and assigning it to the variable `msg`. 

What I’m saying, is to just make it a string in the first place.

Reply:
agreed with @davidnagli, if parcel is aiming to be fast, we can't spent time on stringify'ing object, even with native `JSON.stringify `

Reply:
@davidnagli It's a mistake to optimize prematurely at the expense of code quality.

There are other reasons to prefer the JSON.stringify version. For example, webstorm IDE support will be able to detect the keys and values being used, but can't for string literals. When refractoring code, typos become harder to detect, since you don't get syntax highlighting. And if a refactor wants to interpolate some data into the object, you'll have to revert it to JSON.stringify anyway.

That said, yeah, you could do `{type: 'reload'}` if you really wanted to, but there is zero chance that a reload emission would impact performance in any measurable way.

Reply:
Yeah, JSON.stringify is not going to affect perf. leave it.